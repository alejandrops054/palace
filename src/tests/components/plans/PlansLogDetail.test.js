/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import PlansLogDetail from '../../../components/plans/PlansLogDetail'

describe('PlansLogDetail component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<PlansLogDetail />)
        expect(wrapper).toMatchSnapshot();
    })

})