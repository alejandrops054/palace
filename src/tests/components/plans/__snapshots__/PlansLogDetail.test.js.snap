// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlansLogDetail component debe mostrar snapshot 1`] = `
<Fragment>
  <Card
    tag="div"
  >
    <CardBody
      tag="div"
    >
      <LogDetail
        column={
          Array [
            Object {
              "dataIndex": "order",
              "key": "order",
              "title": "Order",
              "width": "1%",
            },
            Object {
              "dataIndex": "fieldname",
              "key": "fieldname",
              "title": "Field Name",
              "width": "1%",
            },
            Object {
              "dataIndex": "original",
              "key": "original",
              "title": "Original",
              "width": "1%",
            },
            Object {
              "dataIndex": "change",
              "key": "change",
              "title": "Change",
              "width": "1%",
            },
          ]
        }
        data={
          Array [
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
            Object {
              "change": "",
              "fieldname": "",
              "order": "",
              "original": "",
            },
          ]
        }
        title="Plan Log Detail"
      />
    </CardBody>
  </Card>
</Fragment>
`;
